content/docs/atom-guide/dev-guide/shaders/azsl/_index.md

```diff
@@ -445,24 +445,24 @@ For StructuredBuffer&lt;MyStruct&gt; case you'll get:
   
```

Comment:
> For ConstantBuffer&lt;MyStruct&gt; case you'll get these offsets:  
> ...
> For StructuredBuffer&lt;MyStruct&gt; case you'll get:  
> ...

These don't need to be in a code block. Consider the suggestions below to remove them from the code block and list the offsets in bullet points. 

1) Make the following edits for lines 438 to 442:
   - Close the code block (remove `/` when you copy/paste).
```
/```

`ConstantBuffer<MyStruct>` produces the following offsets:  

- `float m_data`: Offset = 0
- `float4 m_arr[2]`: Offset = 16
```

2) Make the following edits for lines 444 to 447:

```
`StructuredBuffer<MyStruct>` produces the following offsets:  

- `float m_data`: Offset = 0
- `float4 m_arr[2]`: Offset = 4
```

Reply:
Fixed.